[2025-03-02T01:11:14.056+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-02T01:11:14.063+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: StockPrice_ETL.load_to_snowflake manual__2025-03-02T01:11:10.947621+00:00 [queued]>
[2025-03-02T01:11:14.066+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: StockPrice_ETL.load_to_snowflake manual__2025-03-02T01:11:10.947621+00:00 [queued]>
[2025-03-02T01:11:14.067+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-03-02T01:11:14.072+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): load_to_snowflake> on 2025-03-02 01:11:10.947621+00:00
[2025-03-02T01:11:14.083+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1053) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-02T01:11:14.085+0000] {standard_task_runner.py:72} INFO - Started process 1056 to run task
[2025-03-02T01:11:14.082+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'StockPrice_ETL', 'load_to_snowflake', 'manual__2025-03-02T01:11:10.947621+00:00', '--job-id', '67', '--raw', '--subdir', 'DAGS_FOLDER/hw5.py', '--cfg-path', '/tmp/tmptpmfwl3l']
[2025-03-02T01:11:14.089+0000] {standard_task_runner.py:105} INFO - Job 67: Subtask load_to_snowflake
[2025-03-02T01:11:14.113+0000] {task_command.py:467} INFO - Running <TaskInstance: StockPrice_ETL.load_to_snowflake manual__2025-03-02T01:11:10.947621+00:00 [running]> on host b90f03c6f926
[2025-03-02T01:11:14.163+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='StockPrice_ETL' AIRFLOW_CTX_TASK_ID='load_to_snowflake' AIRFLOW_CTX_EXECUTION_DATE='2025-03-02T01:11:10.947621+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-02T01:11:10.947621+00:00'
[2025-03-02T01:11:14.164+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-02T01:11:14.169+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-02T01:11:14.170+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-02T01:11:14.171+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-02T01:11:14.171+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-02T01:11:14.778+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-03-02T01:11:14.897+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-03-02T01:12:16.897+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-03-02T01:12:16.904+0000] {logging_mixin.py:190} INFO - Data successfully inserted into Snowflake.
[2025-03-02T01:12:16.905+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-02T01:12:16.937+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-02T01:12:16.937+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=StockPrice_ETL, task_id=load_to_snowflake, run_id=manual__2025-03-02T01:11:10.947621+00:00, execution_date=20250302T011110, start_date=20250302T011114, end_date=20250302T011216
[2025-03-02T01:12:16.997+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-02T01:12:17.016+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-02T01:12:17.017+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
